src/content/en/updates/2019/12/devtools.md

```diff
@@ -0,0 +1,232 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Support for let redeclarations in the Console, improved WebAssembly debugging, and more.
+
+{# wf_updated_on: 2019-12-05 #}
+{# wf_published_on: 2019-12-05 #}
+{# wf_tags: chrome80, devtools, devtools-whatsnew #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: Support for let redeclarations in the Console, improved WebAssembly debugging, and more. #}
+{# wf_blink_components: Platform>DevTools #}
+
+# What's New In DevTools (Chrome 80) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+## Support for `let` redeclarations in the Console {: #let }
+
+The Console now supports redeclarations of `let` statements. The inability to redeclare
+`let` statements was a common annoyance for web developers who use the Console to experiment with
+new JavaScript code.
+
+<aside class="warning">
+  Redeclaring a <code>let</code> statement in a script will still cause a
+  <code>SyntaxError</code>. <code>let</code> redeclarations only work in the Console.
+</aside>
+
+Previously, when redeclaring a local variable with `let`, the Console would throw an error:
+
+<figure>
+  <img src="../../images/2019/12/letbefore.png"
+       alt="A screenshot of the Console in Chrome 78 showing that the let redeclaration fails."/>
+</figure>
+
+Now, the Console allows the redeclaration:
+
+<figure>
+  <img src="../../images/2019/12/letafter.png"
+       alt="A screenshot of the Console in Chrome 80 showing that the let redeclaration succeeds."/>
+</figure>
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/53a8548ce03957e7704ae88b780dfa019bf967e0 #}
+
+Chromium issue [#1004193](https://crbug.com/1004193)
+
+## Improved WebAssembly debugging {: #webassembly }
+
+DevTools has started to support the [DWARF Debugging Standard](http://dwarfstd.org/),
+which means that you can now step over code, set breakpoints, and resolve stack traces in your
+source languages within DevTools.
+Check out [Improved WebAssembly debugging in Chrome DevTools](/web/updates/2019/12/webassembly)
+for the full story.
+
+<figure>
+  <img src="../../images/2019/12/dwarf.png"
```

Comment:
This screenshot looks different than all the others. Can we get one in the same style? @rreverser, do you have a demo link handy that can be used to reproduce this?

Reply:
Note that this is a separate post, and screenshots within it are consistent... but yeah, they're not from macOS like the usual screenshots seem to be.

I don't have a demo link. I feel like I've had to answer this question quite frequently lately 😅 unfortunately, it's not possible to provide one, because DWARF uses local file paths and so debugging would work only on the same machine, at least until we implement remapping UI or something.

But, given that this is in Chromium and if you have Mac, I'm happy to sit down with you and remake the screenshots.

Reply:
Ah, in this context it's actually not a separate post, so yeah, def need to change.

Reply:
Thanks for the updated screenshot @mathiasbynens & @RReverser